bonnie-businessrules/src/main/java/com/cgi/hexagon/businessrules/user/UserStorage.java

```diff
@@ -10,4 +12,8 @@ public interface UserStorage {
 
     boolean save(User user);
 
+    List<User> getAllUsers();
+
+    User getUserByUsername(String username);
```

Comment:
we don't user User in the name of methods in UserStorage interace.
In the OrderStorage these methods are 
`getAllUsers() -> findAll()` and `getUserByUserName() -> findByUserName() `

Reply:
Going to fix. Thanks!